kythe/go/extractors/config/extractor.go

```diff
@@ -60,50 +60,76 @@ type Repo struct {
 	// TODO(#156): this should be removed as soon as we refactor code to no
 	// longer use docker-in-docker.
 	TempRepoDir string
+
+	// An optional path that dicates where temporary output should go before
```

Comment:
"dictates" sp. 

But I'd simplify:  `// An optional temporary directory path to write output before copying to OutputPath.`

Maybe say something about why this helps with d-in-d?

Reply:
Also: What happens if it's not set.